src/main/kotlin/model/OASys.kt

```diff
@@ -1,15 +1,31 @@
 package uk.gov.justice.hmpps.architecture
 
+import com.structurizr.model.Container
 import com.structurizr.model.Model
 import com.structurizr.model.SoftwareSystem
 import com.structurizr.view.ViewSet
 
 class OASys private constructor() {
   companion object : HMPPSSoftwareSystem {
     lateinit var system: SoftwareSystem
+    lateinit var assessmentsApi: Container
 
     override fun defineModelEntities(model: Model) {
-      system = model.addSoftwareSystem("Offender Assessment System", "(OASys) Assesses the risks and needs of offenders")
+      system = model.addSoftwareSystem("Offender Assessment System", "(OASys) Assesses the risks and needs of offenders").apply {
+        setUrl("https://dsdmoj.atlassian.net/wiki/spaces/~474366104/pages/2046820357/OASys+Overview")
+      }
+
+      val oracleApp = system.addContainer("Case Management System", "Assesses the risks and needs of offenders", "Oracle APEX")
+
+      assessmentsApi = system.addContainer("Assessments API", "REST access to the OASYS Oracle DB offender assessment information", "Java + Spring Boot").apply {
+        setUrl("https://github.com/ministryofjustice/offender-assessments-api")
+        APIDocs("http://assessments-api-dev.ngm7p4zgik.eu-west-2.elasticbeanstalk.com/swagger-ui.html").addTo(this)
```

Comment:
❤️

Reply:
Do we want to use the `dev` URL for the swagger docs? Is there a risk production will lag behind changes to these docs?

Reply:
You're right. Ideally, I'd like to use links from pre-production or production.

This link is the only one I could find for this service 😅